### PR TITLE
Adcm-2936 add ui tests for ldap users and groups

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/app/page/admin/page.py [hunk 1]
```diff
@@ -358,6 +382,38 @@ def click_delete_button(self):
         self.find_and_click(DeleteDialog.yes)
         self.wait_element_hide(DeleteDialog.body)
 
+    @allure.step('Get group {group_name}')
+    def get_group_by_name(self, group_name: str):
+        """Get group by name"""
+        for group in self.table.get_all_rows():
+            if group_name in group.text:
+                return group
```
Comment (Sealwing) [inline tests/ui_tests/app/page/admin/page.py:390]: raise exception (AssertionError) otherwise?

File: tests/ui_tests/test_admin_page.py [hunk 2]
```diff
@@ -355,6 +355,17 @@ def test_change_admin_password(self, users_page: AdminUsersPage):
         with allure.step('Check login was successful'):
             AdminIntroPage(users_page.driver, users_page.base_url).wait_page_is_opened(timeout=5)
 
+    @pytest.mark.usefixtures("configure_adcm_ldap_ad")
+    def test_change_ldap_user(self, users_page: AdminUsersPage, ldap_user_in_group):
```
Comment (Sealwing) [inline tests/ui_tests/test_admin_page.py:359]: change or check? I don't see change

File: tests/ui_tests/test_admin_page.py [hunk 3]
```diff
@@ -499,6 +510,19 @@ def test_delete_group_from_groups_page(self, app_fs):
         with allure.step('Check that group has been deleted'):
             assert len(page.table.get_all_rows()) == 0, "There should be 0 groups"
 
+    @pytest.mark.usefixtures("configure_adcm_ldap_ad")
+    def test_change_ldap_group(self, app_fs, ldap_user_in_group):
```
Comment (Sealwing) [inline tests/ui_tests/test_admin_page.py:514]: change or check? I don't see change

**Review by Sealwing — CHANGES_REQUESTED** [review]

File: tests/ui_tests/test_admin_page.py (hunk 3 above)
Comment (annademina) [reply tests/ui_tests/test_admin_page.py]: changed to disabled

File: tests/ui_tests/test_admin_page.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/test_admin_page.py]: changed to disabled

File: tests/ui_tests/app/page/admin/page.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/app/page/admin/page.py:389]: yes, added

**Review by Sealwing — DISMISSED** [review]

**Review by kuhella — COMMENTED** [review]
Need to add step  where ANY user might be added to local group according to https://tracker.yandex.ru/ADCM-3014

**Review by Sealwing — APPROVED** [review]

**Pull request merged**